runtime.md

```diff
@@ -121,11 +121,18 @@ When the process in the container is stopped, irrespective of it being as a resu
 This operation MUST generate an error if it is not provided the container ID.
 Attempting to delete a container that does not exist MUST generate an error.
 Attempting to delete a container whose process is still running MUST generate an error.
-Deleting a container MUST delete the resources that were created during the `create` step.
-Note that resources associated with the container, but not created by this container, MUST NOT be deleted.
 Once a container is deleted its ID MAY be used by a subsequent container.
 
+Deleting a container MUST delete the resources that were created during the `create` step.
+Resources associated with the container, but not created by this container, MUST NOT be deleted.
+
+On Linux, [`mounts`](config.md#mounts), [`root`](config.md#root), [`linux.devices`](config-linux.md#devices), and other filesystem changes belong to the container which created the [container mount namespace][container-namespace3].
+If a container joins an existing mount namespace and applies those settings, deleting the container MUST NOT reverted the settings.
+Similarly [`hostname`](config.md#hostname) changes belong to the container which created the container UTS namespace.
+[`linux.resources`](config-linux.md#control-groups) settings belong to the container which created the associated cgroup.
```

Comment:
And I've just pushed 52a5199 → eeacccd adding a new mount namespace to the UTS example, otherwise the (required) `root.path` entry would pivot the host mount namespace.